<<<<<<< 61ddbc132891cf9533228634c5d59b0c5d30fb7b
<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FourthDialog</class>
 <widget class="QDialog" name="FourthDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>111</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-image: url(:/main/images/biankuang_5.png);
background-color: rgb(170, 170, 127);</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>70</y>
     <width>101</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>发起挑战</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
border-image: url(:/main/images/back_normal_def1ault.png);
font: 14pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>课程</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>30</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/main/images/back_normal_defaul11t.png);</string>
   </property>
  </widget>
  <widget class="QLabel" name="flash">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>26</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/main/images/backgr1ound3.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>61</x>
     <y>20</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
border-image: url(:/main/images/back_normal_def1ault.png);
font: 14pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>请求已经发送</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>61</x>
     <y>60</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
border-image: url(:/main/images/back_normal_def1ault.png);
font: 14pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>等待对方同意</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
=======
<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FourthDialog</class>
 <widget class="QDialog" name="FourthDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>111</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-image: url(:/main/images/biankuang_5.png);
background-color: rgb(170, 170, 127);</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>70</y>
     <width>101</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>发起挑战</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
border-image: url(:/main/images/back_normal_def1ault.png);
font: 14pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>课程</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>30</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/main/images/back_normal_defaul11t.png);</string>
   </property>
  </widget>
  <widget class="QLabel" name="flash">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>26</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/main/images/backgr1ound3.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>61</x>
     <y>20</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
border-image: url(:/main/images/back_normal_def1ault.png);
font: 14pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>请求已经发送</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>61</x>
     <y>60</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
border-image: url(:/main/images/back_normal_def1ault.png);
font: 14pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>等待对方同意</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
>>>>>>> final version
